scripts/assets/file_content_validation_checks.textproto

```diff
@@ -47,6 +47,7 @@ file_content_checks {
   exempted_file_name: "app/src/main/res/layout-sw600dp-land/profile_progress_header.xml"
   exempted_file_name: "app/src/main/res/layout-sw600dp-port/profile_progress_header.xml"
   exempted_file_name: "app/src/main/res/values/styles.xml"
+  exempted_file_name: "app/src/main/res/values/themes.xml"
```

Comment:
Which line is failing the check that leads to needing this exemption?

Reply:
@BenHenning Sorry. We don't actually need those changes. Reverted it.